app/utils.py

```diff
@@ -433,6 +433,20 @@ def check_time_format(field_val, row_index):
 							db.session.add(timeslot)
 							timeslots.append(timeslot)
 
+				# Checks database to see if identical resource exists
+				duplicate = FoodResource.query.filter_by(
+					name = name, 
+					url = website,
+					description = description,
+					are_hours_available = are_hours_available,
+					is_for_family_and_children = is_for_family_and_children,
+					is_for_seniors = is_for_seniors,
+					is_wheelchair_accessible = is_wheelchair_accessible,
+					is_accepts_snap = is_accepts_snap).first()
+
+				if duplicate:
+					make_error("Identical resource already exists in database.", i)
+
```

Comment:
You might note in the error which resource is a duplicate. For example, "Identical resource (Clark Park) already exists in the database."